DQM/GEM/interface/GEMDQMBase.h

```diff
@@ -578,7 +578,7 @@ inline bool GEMDQMBase::checkRefs(const std::vector<T *> &refs) {
   return true;
 }
 
-// The 'get...' functions in the below are borrwed from DQMOffline/Muon/interface/GEMOfflineDQMBase.h
+// The 'get...' functions in the below are borrwed from DQM/GEM/interface/GEMOfflineDQMBase.h
```

Comment:
Please revert (ok, it is a comment line, but let have consistent comments in the release)